### PR TITLE
Add link below tx pending state

### PR DESCRIPTION
**Pull request by AndrejZelonka**
Adding link below tx pending state.

image

### CODE REVIEW

**Review by PeterBenc — APPROVED** [review]
LGTM, thanks

**Review by PeterBenc — COMMENTED** [review]
Sorry, approved too soon and forgot to check it really, I think it looks a bit "not great". 

The whole page jumps when the link appears and creates a space between the tabs and the content

image

Not sure if this is supposed to look like this on mobile devices
image

File: app/frontend/components/pages/dashboard/walletOperationStatus.tsx [hunk 1]
```diff
@@ -24,7 +24,21 @@ export const WalletOperationStatusType = () => {
...
+        <Fragment>
+          <PinkSpinner message={'Transaction pending...'} />
+          <p className="loading-helper-text">
+            Taking too long? Read why{' '}
+            <a
+              href="https://github.com/vacuumlabs/adalite/wiki/Troubleshooting#my-transaction-has-been-pending-for-a-long-time--what-should-i-do"
+              target="_blank"
+            >
+              here
+            </a>
```
Comment (PeterBenc) [inline app/frontend/components/pages/dashboard/walletOperationStatus.tsx:37]: Just a suggestion, If we wanted to make it cool, we could start a `timer` on `useEffect` and show the message lets say after 10 seconds.

File: app/frontend/components/pages/dashboard/walletOperationStatus.tsx [hunk 2]
```diff
@@ -25,19 +46,23 @@ export const WalletOperationStatusType = () => {
...
-              target="_blank"
-            >
-              here
-            </a>
-            .
-          </p>
-        </Fragment>
+        <PinkSpinner
+          message={'Transaction pending...'}
+          helperText={
```
Comment (AndrejZelonka) [inline app/frontend/components/pages/dashboard/walletOperationStatus.tsx:51]: Seems easier to add helper text absolutely to PinkSpinner as it has hardcoded margin values for responsive designs.

File: app/frontend/components/pages/dashboard/walletOperationStatus.tsx [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
-const PinkSpinner = ({message}) => (
-  <div className="header-message">
-    {message.split('\n').map((line, i) => (
-      <p className="loading-message black" key={i}>
-        {line}
-      </p>
-    ))}
-    <div className="spinner pink">
-      <span />
+const DelayRendering = ({children}) => {
```
Comment (AndrejZelonka) [inline app/frontend/components/pages/dashboard/walletOperationStatus.tsx:8]: Rather then creating condition whitin `PinkSpinner` `useEffect` I've created new component for delay rendering, Seems more elegant imho

File: app/public/css/768-1024px.css [hunk 4]
```diff
@@ -32,6 +32,10 @@
     margin: 16px 0px -42px;
   }
 
+  .helper-text-wrapper {
+    margin-bottom: -37px;
```
Comment (PeterBenc) [inline app/public/css/768-1024px.css:36]: Nitpick: I think the policy is that the px should be multiple of 4 or at least an even number. If it looks worse with an even number then feel free to ignore this. Although, I think the "Taking too long" text could be a pixel or two close to the helper text and farther from the bottom.

**Review by PeterBenc — APPROVED** [review]
LGTM

**Pull request merged**